Makefile

```diff
@@ -5,6 +5,7 @@ GOBUILD=go build
 GOPATH=$(TARGET_DIR):$(TARGET_DIR)/vendor:$(CURPATH)/cmd
 
 DOCKER_OPTS=
+IMAGE_BUILD=docker build
```

Comment:
Should this be in quotes?

Reply:
@ewolinetz per [1] not required

[1] https://ftp.gnu.org/old-gnu/Manuals/make-3.79.1/html_chapter/make_6.html#SEC64